packages/story-editor/src/components/elementLink/output.js

```diff
@@ -31,14 +31,16 @@ function WithLink({ element, children, ...rest }) {
   if (!link?.url?.length) {
     return children;
   }
+  const rel = link.rel ? link.rel.join(' ') : 'noreferrer';
```

Comment:
We need to always add `noreferrer`

So could be `noreferrer nofollow sponsored` for instance.

Reply:
Fixed.